Apps/WebClient/src/ClientApp/src/components/DependentCardComponent.vue

```diff
@@ -1341,6 +1447,119 @@ export default class DependentCardComponent extends Vue {
+                    :disabled="isExpired"
+                    no-body
+                    :data-testid="`clinical-document-tab-${dependent.ownerId}`"
+                    @click="fetchClinicalDocuments"
+                >
+                    <template #title>
+                        <div
+                            :data-testid="`clinical-docuemnt-tab-title-${dependent.ownerId}`"
+                        >
+                            Clinical Docs
+                        </div>
+                    </template>
+                    <div class="p-3">
+                        <b-spinner v-if="isLoading" class="mt-3" />
+                        <div
+                            v-if="!isLoading && clinicalDocuments.length === 0"
+                            :data-testid="`clinical-document-no-records-${dependent.ownerId}`"
+                        >
+                            No records found.
+                        </div>
+                    </div>
+                    <b-table-simple
+                        v-if="!isLoading && clinicalDocuments.length > 0"
+                        small
+                        striped
+                        borderless
+                        :items="clinicalDocuments"
+                        class="w-100 mb-0"
+                        aria-describedby="Clinical Docs Results"
+                        :data-testid="`clinical-document-table-${dependent.ownerId}`"
+                    >
+                        <b-thead>
+                            <b-tr>
+                                <b-th class="align-middle">Date</b-th>
+                                <b-th class="align-middle">Title</b-th>
+                                <b-th
+                                    class="d-none d-md-table-cell align-middle"
+                                >
+                                    Document Type
+                                </b-th>
+                                <b-th
+                                    class="d-none d-md-table-cell align-middle"
+                                >
+                                    Discipline
+                                </b-th>
+                                <b-th
+                                    class="d-none d-md-table-cell align-middle"
+                                >
+                                    Facility Name
+                                </b-th>
+                                <b-th class="align-middle">Report</b-th>
+                            </b-tr>
+                        </b-thead>
+                        <b-tbody>
+                            <b-tr
+                                v-for="(row, index) in clinicalDocuments"
+                                :key="index"
+                            >
+                                <b-td
+                                    :data-testid="`clinical-document-service-date-${dependent.ownerId}-${index}`"
+                                    class="align-middle"
```

Comment:
Should add `text-nowrap` class per #4466.

Reply:
Fixed